docs/csharp/whats-new/csharp-11.md

```diff
@@ -0,0 +1,84 @@
+---
+title: What's new in C# 11 - C# Guide
+description: Get an overview of the new features coming in C# 11.
+ms.date: 03/04/2022
+---
+# What's new in C# 11
+
+Beginning with the .NET 6.0.200 SDK or Visual Studio 2022 version 17.1, preview features in C# are available for you to try.
+
+> [!IMPORTANT]
+> These are currently preview features. You must [set `<LangVersion>` to `preview`](../language-reference/compiler-options/language.md#langversion) to enable these features. Any feature may change before its final release. These features may not all be released in C# 11. Some may remain in a preview phase for longer based on feedback on the feature.
+
+The following features are available in the 6.0.200 version of the .NET SDK. They are available in Visual Studio 2022 version 17.1.
+
+- [Generic attributes](#generic-attributes).
+- [static abstract members in interfaces](#static-abstract-members-in-interfaces).
+
+You can download the latest .NET 6 SDK from the [.NET downloads page](https://dotnet.microsoft.com/download). You can also download [Visual Studio 2022](https://visualstudio.microsoft.com/vs/), which includes the .NET 6 SDK.
+
+## Generic attributes
+
+You can declare a [generic class](../programming-guide/generics/generic-classes.md) whose base class is <xref:System.Attribute?displayProperty=fullName>. This provides a more convenient syntax for attributes that require a <xref:System.Type?displayProperty=nameWithType> parameter. Previously, you'd need to create an attribute that takes a `Type` as its constructor parameter:
+
+```csharp
```

Comment:
When people are skimming, they may have trouble distinguishing old and new code. This will be an ongoing issue. Is it worth a consistent comment on either the old or the new code to help people keep that straight?

Reply:
Rather than "old" and "new", I chose "Before <version>" and "<version feature was introduced> feature"